### DISCUSSION TITLE
Question about headers in playlist view

### DISCUSSION THREAD

**Discussion by bathory-0** [TT-ReBORN/Georgia-ReBORN#108] [Q&A] [answered]
Hi, I'm asking for some guidance about where can I edit the header portion of the playlist view here:
![screenshot_857](https://user-images.githubusercontent.com/58579324/228900718-64003e13-d6ce-4f87-9b95-cadf6dc54579.jpg)

My idea is to edit the portion of the header that shows "genre | coding | tracks | time" but I couldn't do it in the Grouping>Manage profiles section, so I think it should be in some .js file.

I would be very grateful if someone could help me in this :)

**Comment by TT-ReBORN** [comment, marked as answer]
Hi @bathory-0,

[if you're using the latest Beta version from here](https://github.com/TT-ReBORN/Georgia-ReBORN/discussions/74#discussion-4190799), you can do that via top menu Options > Playlist > Album header > Customize header info. This is a shortcut to write your custom pattern in your georgia-reborn-config.jsonc file in section 
`"settings"`:
`"playlistCustomHeaderInfo": "", // You can use your own custom pattern for the playlist header info`
which you can also write it directly in your config file.

You can also do the same for title row -> Options > Playlist > Track row > Customize track row
`"settings"`: 
`"playlistCustomTitle": "", // You can use your own custom title pattern for the playlist row`

If both are blank again, it will use the default patterns.

-TT
